src/modules/inventory/index.ts

```diff
@@ -18,4 +18,12 @@ export class Inventory {
     });
     return response.data;
   }
+
+  public async getAllProduct(): Promise<CreatedProduct> {
```

Comment:
```suggestion
  public async getAllProducts(): Promise<CreatedProduct> {
```
Please update all corresponding references.